src/selectors/entities/channels.js

```diff
@@ -566,6 +566,29 @@ export const canManageChannelMembers: (GlobalState) => boolean = createSelector(
     }
 );
 
+// Determine if the user has permissions to manage members in at least one channel of the current team
+export const canManageAnyChannelMembers: (GlobalState) => boolean = createSelector(
```

Comment:
Maybe change the name to `canManageAnyChannelMembersInCurrentTeam`?

Reply:
👍 done